README.md

```diff
@@ -4,6 +4,77 @@ JS Client library to interact with Supabase Storage.
 
 - Documentation: https://supabase.io/docs/reference/javascript/storage-createbucket
 
+## Quick Start Guide
+
+### Installing the module
+
+``` bash
+npm install @supabase/storage-js
+```
+
+### Connecting to the storage backend
+
+``` bash
+# .env
+SUPABASE_STORAGE_URL=https://<project_ref>.supabase.co/storage/v1
+SUPABASE_SERVICE_KEY=<service_role>
+```
+
+``` js
+// index.hs
+import { SupabaseStorageClient } from '@supabase/storage-js'
+import 'dotenv/config'
+
+const STORAGE_URL = process.env.SUPABASE_STORAGE_URL;
+const SERVICE_KEY = process.env.SUPABASE_SERVICE_KEY; //! service key, not anon key
+
+const storage_client = new SupabaseStorageClient(
```

Comment:
camelCase for consistency with documentation elsewhere? Here and below.

Reply:
Uh oh. Bad copy paste job. My bad :))

Will fix

Reply:
[Fixed](https://github.com/supabase/storage-js/commit/0af5169a884d5d9f0bd30be8d79526f263c46d0e)